templates/stub.logsearch-for-cloudfoundry.yml

```diff
@@ -58,3 +69,11 @@ properties:
   syslog:
     host: (( grab jobs.ingestor.networks.[0].static_ips.[0] ))
     port: 5514
+  smoke_tests:
+    api: api.example.com
```

Comment:
Can we steal these values from another place, `cf-kibana` properties for example?